isthmus/src/main/java/io/substrait/isthmus/sql/SubstraitSqlToCalcite.java

```diff
@@ -0,0 +1,163 @@
+   *     the SQL statement
+   * @return a {@link RelRoot} corresponding to the given SQL statement
+   * @throws SqlParseException if there is an error while parsing the SQL statement
+   */
+  public static RelRoot convertQuery(String sqlStatement, Prepare.CatalogReader catalogReader)
+      throws SqlParseException {
+    SqlValidator validator = new SubstraitSqlValidator(catalogReader);
+    return convertQuery(sqlStatement, catalogReader, validator, createDefaultRelOptCluster());
+  }
+
+  /**
+   * Converts a SQL statement to a Calcite {@link RelRoot}.
+   *
+   * @param sqlStatement a SQL statement
+   * @param catalogReader the {@link Prepare.CatalogReader} for finding tables/views referenced in
+   *     the SQL statement
+   * @param validator the {@link SqlValidator} used to validate SQL statements. Allows for
+   *     additional control of SQL functions and operators via {@link
+   *     SqlValidator#getOperatorTable()}
+   * @param cluster the {@link RelOptCluster} used when creating {@link RelNode}s during statement
+   *     processing. Calcite expects that the {@link RelOptCluster} used during statement processing
+   *     is the same as that used during query optimization.
+   * @return {@link RelRoot} corresponding to the given SQL statement
+   * @throws SqlParseException if there is an error while parsing the SQL statement string
+   */
+  public static RelRoot convertQuery(
+      String sqlStatement,
+      Prepare.CatalogReader catalogReader,
+      SqlValidator validator,
+      RelOptCluster cluster)
+      throws SqlParseException {
+    List<SqlNode> sqlNodes = SubstraitSqlStatementParser.parseStatements(sqlStatement);
+    if (sqlNodes.size() != 1) {
+      throw new IllegalArgumentException(
+          String.format("Expected one statement, found: %d", sqlNodes.size()));
+    }
+    List<RelRoot> relRoots = convert(sqlNodes, catalogReader, validator, cluster);
+    // as there was only 1 statement, there should only be 1 root
+    return relRoots.get(0);
+  }
+
+  /**
+   * Converts one or more SQL statement to a List of {@link RelRoot}, with one {@link RelRoot} per
+   * statement.
+   *
+   * @param sqlStatements a string containing one or more SQL statements
+   * @param catalogReader the {@link Prepare.CatalogReader} for finding tables/views referenced in
+   *     the SQL statement
+   * @return a list of {@link RelRoot}s corresponding to the given SQL statements
+   * @throws SqlParseException if there is an error while parsing the SQL statements
+   */
+  public static List<RelRoot> convertQueries(
+      String sqlStatements, Prepare.CatalogReader catalogReader) throws SqlParseException {
+    SqlValidator validator = new SubstraitSqlValidator(catalogReader);
+    return convertQueries(sqlStatements, catalogReader, validator, createDefaultRelOptCluster());
+  }
+
+  /**
+   * Converts one or more SQL statement to a List of {@link RelRoot}, with one {@link RelRoot} per
```

Comment:
minor: I noticed in `SubstraitSqlStatementParser` you were using the plural here:

```suggestion
   * Converts one or more SQL statements to a List of {@link RelRoot}, with one {@link RelRoot} per
```